.azure/OneBranch.Publish.yml

```diff
@@ -27,7 +27,7 @@ parameters:
     type: object
     default:
     - name: RPM
-      destinations:
+      destination:
```

Comment:
These need to be `destinations` with the `s`. that's what is used down below.

Reply:
I'm afraid we're not communicating very well.  The instructions above were to turn it from destionation**s** to destination,  There is a typo in what was there earlier, the extra o someone left in there, but the fix suggested was to strip off the s at the end.  I didn't even see the o, just the removal of the s.  My comment that they were all like that (plural) coupled with your 'can you fix them all' comment made me think you wanted it to just say destination (singular) everywhere.  In any event, now that I've caught up, proper fix inbound.